src/components/tables/QueueTable.tsx

```diff
@@ -217,7 +217,7 @@ const QueueTable = () => {
           type: "text-short",
           modifier: (data) => (
             <Text>
-              {`~${calculateReadingTime(Number(data.contentTotalWords))}`}
+              {`${calculateReadingTime(Number(data.contentTotalWords))}`}
```

Comment:
Lets keep the tilda sign (~). We want to show that its an approximate time not absolute